dockstore-common/src/main/java/io/dockstore/common/PipHelper.java

```diff
@@ -35,7 +35,9 @@ public static String convertSemVerToAvailableVersion(String semVerString) {
             semVerString = "9001.9001.9001";
         }
         Version semVer = Version.valueOf(semVerString);
-        if (semVer.greaterThanOrEqualTo(Version.valueOf("1.14.0"))) {
+        if (semVer.greaterThanOrEqualTo(Version.valueOf("1.15.0"))) {
```

Comment:
Pending CLI pr, I think this is the CLI version and we don't know yet if this is backwards compatible

Reply:
I don't follow, but there's a good chance I'm missing something... This is the CLI version, only the 1.15 CLI will hit this. Shouldn't this just work?

Reply:
Elaborating, we've released a 1.15 cli. After this change, existing users with a 1.15 cli will get the new requirements instead of the old 1.14.0 requirements from our webservice. 

At the time the comment was written, I didn't know if the new dependencies are backwards compatible. 
Afterwards, we found out that they actually aren't backwards compatible for one reason,. they will fail on Ubuntu 20.04.  There may be other reasons that our 1.15.0 CLI code is not compatible with these requirements, we need to run cli tests with the proposed webservice to find  out.